boards/remote-reva/include/board.h

```diff
@@ -8,9 +8,7 @@
  */
 
 /**
- * @defgroup    boards_reva RE-Mote Revision A
- * @ingroup     boards
- * @brief       Support for the RE-Mote board Revision A
+ * @ingroup     boards_reva
```

Comment:
The original group name of this board could have been more specific and then consistent with other `remote` boards: boards_remote-reva